tests/arp/test_stress_arp.py

```diff
@@ -0,0 +1,168 @@
+
+    pytest_require(garp_enabled, 'Gratuitous ARP not enabled for this device')
+    ptf_intf_ipv4_hosts = genrate_ipv4_ip()
+    ptf_intf_ipv4_hosts = list(ptf_intf_ipv4_hosts)[1:arp_avaliable + 1]
+    _, _, intf1_index, _, = intfs_for_test
+
+    loop_times = LOOP_TIMES_LEVEL_MAP[normalized_level]
+
+    while loop_times > 0:
+        loop_times -= 1
+        add_arp(ptf_intf_ipv4_hosts, intf1_index, ptfadapter)
+
+        pytest_assert(wait_until(20, 1, 0, lambda: get_fdb_dynamic_mac_count(duthost) >= arp_avaliable),
+                      "ARP Table Add failed")
+
+        clear_dut_arp_cache(duthost)
+        fdb_cleanup(duthost)
+
+        time.sleep(5)
+
+def generate_link_local_addr(mac):
+    parts = mac.split(":")
+    parts.insert(3, "ff")
+    parts.insert(4, "fe")
+    parts[0] = "{:x}".format(int(parts[0], 16) ^ 2)
+
+    ipv6Parts = []
+    for i in range(0, len(parts), 2):
+        ipv6Parts.append("".join(parts[i:i+2]))
+    ipv6 = "fc02:1000::{}".format(":".join(ipv6Parts))
+    return ipv6
+
+
+def packets_for_test(duthost, config_facts, tbinfo, ip_and_intf_info, fake_src_mac, fake_src_addr):
+    _, _, src_addr_v6, _, _ = ip_and_intf_info
+    fake_src_mac = fake_src_mac
+    vlans = config_facts['VLAN']
+    topology = tbinfo['topo']['name']
+    dut_mac = ''
+    for vlan_details in vlans.values():
+        if 'dualtor' in topology:
+            dut_mac = vlan_details['mac'].lower()
+        else:
+            dut_mac = duthost.shell('sonic-cfggen -d -v \'DEVICE_METADATA.localhost.mac\'')["stdout_lines"][0].decode(
+                "utf-8")
+        break
+
+    tgt_addr = increment_ipv6_addr(src_addr_v6)
+    multicast_tgt_addr = in6_getnsma(inet_pton(socket.AF_INET6, tgt_addr))
+    multicast_tgt_mac = in6_getnsmac(multicast_tgt_addr)
+
+    ns_pkt = Ether(src=fake_src_mac, dst=multicast_tgt_mac)
+    ns_pkt /= IPv6(dst=inet_ntop(socket.AF_INET6, multicast_tgt_addr), src=fake_src_addr)
+    ns_pkt /= ICMPv6ND_NS(tgt=tgt_addr)
+    ns_pkt /= ICMPv6NDOptSrcLLAddr(lladdr=fake_src_mac)
+    logging.info(repr(ns_pkt))
+
+    na_pkt = Ether(src=fake_src_mac, dst=dut_mac)
+    na_pkt /= IPv6(dst=inet_ntop(socket.AF_INET6, multicast_tgt_addr), src=fake_src_addr)
+    na_pkt /= ICMPv6ND_NA(tgt=fake_src_addr, S=1, R=1, O=0)
+    logging.info(repr(na_pkt))
+
+    return ns_pkt, na_pkt
+
+
+def add_nd(duthost, ptfadapter, config_facts, tbinfo, ip_and_intf_info, ptf_intf_index, nd_avaliable):
+    for entry in range(0, nd_avaliable):
+        nd_entry_mac = IntToMac(MacToInt(ARP_SRC_MAC) + entry)
+        fake_src_addr = generate_link_local_addr(nd_entry_mac)
+        ns_pkt, na_pkt = packets_for_test(duthost, config_facts, tbinfo, ip_and_intf_info, nd_entry_mac, fake_src_addr)
+
+        ptfadapter.dataplane.flush()
+        testutils.send_packet(ptfadapter, ptf_intf_index, ns_pkt)
+        duthost.shell(
+            "sudo ip neighbor change {} lladdr {} dev Vlan1000 nud reachable".format(fake_src_addr, nd_entry_mac))
```

Comment:
Add a TODO tag and add a comment explaining why doing this?

Reply:
I change the method to add entries here, so it will be used.